l10n_br_stock_account/tests/test_invoicing_picking.py

```diff
@@ -289,8 +289,8 @@ def test_invoicing_picking_lucro_presumido(self):
         self.assertEqual(
             invoice.partner_id, self.env.ref("l10n_br_base.res_partner_cliente1_sp")
         )
-        # Campo 'Consumidor Final' deve estar vazio
-        assert invoice.ind_final, "Error field ind_final not None"
```

Comment:
@marcelsavegnago o teste foi incluído no PR https://github.com/OCA/l10n-brazil/pull/2941 devido o erro https://github.com/OCA/l10n-brazil/issues/2924 como o campo ind_final está com default True ao chamar algum método que simula onchanges ou pega os valores padrões do campo para a criação de uma account.move o campo acabava indo como True mesmo quando vazio, o teste foi incluído para validar essa situação, porque isso afetava os cálculos dos impostos.

Teve algum erro que levou a isso? Algum LOG?

Já tinha comentando anteriormente que acho que o melhor é que o default desse campo seja None, é ou algum onchange do partner traz esse valor do cadastro ou o usuário deveria informar manualmente para evitar esses casos onde algum método pega o valor default de um campo quando está vazio

Reply:
Boto fé que quando não for do Brasil o valor deveria ser Null. Porém, neste caso estamos lidando com um picking que vai gerar uma fatura e o campo precisa ter valor. Pelo que entendi o valor se é ind_final ou não depende do parceiro e portando apenas busquei garantir que se for alterado algo no parceiro vamos sentir no teste.

fiscal
_onchange_partner_id_fiscal